packages/webrtc-media/package.json

```diff
@@ -19,7 +19,7 @@
   "scripts": {
     "prebuild": "rimraf dist",
     "build": "tsc && node postcompile.mjs && rollup --config",
-    "test": "jest",
+    "test": "npx jest",
```

Comment:
if we go ahead with this, we should probably specify the version as well

Reply:
I have taken a action item for it will fix it with a later PR